src/cli/cli_udp.cpp

```diff
@@ -77,6 +106,27 @@ template <> otError UdpExample::Process<Cmd("bind")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli udp connect
+ * @code
+ * udp connect fdde:ad00:beef:0:bb1:ebd6:ad10:f33 1234
+ * Done
+ * @endcode
+ * @code
+ * udp connect 172.17.0.1 1234
+ * Connecting to synthesized IPv6 address: fdde:ad00:beef:2:0:0:ac11:1
+ * Done
+ * @endcode
+ * @cparam udp connect @ca{ip} @ca{port}
+ * The following parameters are required:
+ * - `ip`: IP address of the peer
+ * - `port`: UDP port number of the peer
```

Comment:
Can you add a period at the end of each first sentence here

IP address of the peer.
UPD port number of the peer.